apps/site/assets/ts/shuttles/components/StationDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement, useState, useEffect } from "react";
+import renderSvg from "../../helpers/render-svg";
+import arrowIcon from "../../../static/images/icon-down-arrow.svg";
+import { ShuttleStop } from "./__shuttles";
+
+interface Props {
+  stations: ShuttleStop[];
+  onSelect: (stationId: string) => void;
+}
+
+const StationDropdown = ({
+  stations,
+  onSelect
+}: Props): ReactElement<HTMLElement> => {
+  const [selectedStationId, setSelectedStationId] = useState("");
+  useEffect(
+    () => {
+      onSelect(selectedStationId);
+    },
+    [onSelect, selectedStationId]
+  );
+
+  return (
+    <div className="c-select-custom__container">
+      <select
+        className="c-select-custom"
+        defaultValue={selectedStationId}
+        onChange={e => setSelectedStationId(e.target.value)}
+      >
+        <option value="">Choose a station</option>
+        {stations.sort().map(s => (
```

Comment:
@thecristen this is why I'm into explicit variable names: sure, you save a few characters, but here `s` is a station, not a shape--which would not be the first thing I thought of having read some of the code above where it does stand for shape. In general, if my collection is named `foos`, I call the iterator `foo`.